### PR TITLE
HIP-584: Copy and adapt implementation for Create Token precompile Part 2

### LINKED ISSUE #5649: HIP-584: Provide implementation for create token precompiles
### Problem

We need default implementations for precompiles related to creation of a new token, so we should copy needed logic from hedera-services into hedera-mirror-web3.

### Solution

Copy precompile logic for the following precompiles:

- TokenCreatePrecompile

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by stoyanov-st**
**Description**:
Copy and adapt Token Create precompile logic from hedera-services

**Related issue(s)**:

Fixes #5649 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-grpc/src/main/java/com/hedera/mirror/grpc/service/TopicMessageServiceImpl.java [hunk 1]
```diff
@@ -163,7 +168,9 @@ private Flux<TopicMessage> missingMessages(TopicContext topicContext, TopicMessa
         TopicMessageFilter newFilter = topicContext.getFilter().toBuilder()
                 .endTime(LongToInstantConverter.INSTANCE.convert(current.getConsensusTimestamp()))
                 .limit(numMissingMessages)
-                .startTime(LongToInstantConverter.INSTANCE.convert(last.getConsensusTimestamp()).plusNanos(1))
+                .startTime(LongToInstantConverter.INSTANCE
+                        .convert(last.getConsensusTimestamp())
+                        .plusNanos(1))
```
Comment (IvanKavaldzhiev) [inline hedera-mirror-grpc/src/main/java/com/hedera/mirror/grpc/service/TopicMessageServiceImpl.java:173]: Remove all of the changes from `hedera-mirror-grpc` module

Comment (stoyanov-st) [reply hedera-mirror-grpc/src/main/java/com/hedera/mirror/grpc/service/TopicMessageServiceImpl.java]: Done

**Review by IvanKavaldzhiev — APPROVED** [review]
LGTM - only need to remove `hedera-mirror-rest` changes

**Comment by steven-sheehy** [comment]
Still hedera-mirror-rest changes and one code smell. Might have to disable spotless git hook temporarily, revert your rest changes, then push, re-enable spotless.

**Comment by stoyanov-st** [comment]
> Still hedera-mirror-rest changes and one code smell. Might have to disable spotless git hook temporarily, revert your rest changes, then push, re-enable spotless.

The formatting issue is handled and the code smell is for cognitive complexity from a method that was copied from services.

**Review by steven-sheehy — DISMISSED** [review]
LGTM

**Pull request merged**